.travis.yml

```diff
@@ -63,7 +63,9 @@ jobs:
       before_install:
         - if [ -z "${TRAVIS_TAG}" ]; then RELEASE_VERSION="SNAPSHOT-${TRAVIS_BUILD_NUMBER}"; else RELEASE_VERSION="${TRAVIS_TAG}";fi
         - echo "Setting version to $RELEASE_VERSION"
-      install: ./gradlew -PbuildVersion=$RELEASE_VERSION assemble packageDocumentation
+        - if [ -z "${TRAVIS_TAG}" ]; then echo "Not release build - skipping docker login."; else ./resources/docker-login.sh;fi
+      # if it is a release build package demo and push agent docker image
+      install: if [ -z "${TRAVIS_TAG}" ]; then ./gradlew -PbuildVersion=$RELEASE_VERSION assemble packageDocumentation; else ./gradlew -PbuildVersion=$RELEASE_VERSION assemble packageDocumentation packageDemo dockerPush;fi
```

Comment:
Please split this in two lines and add the comment before the second

    - ./gradlew -PbuildVersion=$RELEASE_VERSION assemble packageDocumentation
    # Push updated docker images to docker hub in case of a release build
    - if [ -z "${TRAVIS_TAG}" ]; then ./gradlew -PbuildVersion=$RELEASE_VERSION dockerPush;fi